v3/v3_service.go

```diff
@@ -2282,3 +2305,76 @@ func (op Operations) UpdateAddressGroup(ctx context.Context, uuid string, body *
 
 	return op.client.Do(ctx, req, nil)
 }
+
+func (op Operations) CreateRecoveryPlanJob(ctx context.Context, request *RecoveryPlanJobIntentInput) (*RecoveryPlanJobResponse, error) {
```

Comment:
comment header required to be consistent with other functions

Reply:
Added to all methods newly added.